docs/pbc/all/service-point-management/202311.0/unified-commerce/click-collect-domains/click-collect-domains.md

```diff
@@ -0,0 +1,68 @@
+<script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js"></script>
+
+
+## Domains and subdomains
```

Comment:
* What was the reason for the removal of the description part before this header?